src/styles/AnswerButton.scss

```diff
@@ -0,0 +1,23 @@
+.answer {
```

Comment:
Aby dołączyć style musisz w pliku `AnswerButton.scss` dodać `import '../styles/AnswerButton.scss';`. Nazwy które nadałeś są dość ogólne, a ponieważ jeszcze nie skonfigurowaliśmy css modules (i nie wiem czy skonfigurujemy :P), to dobrze byłoby używać jakiegoś prefixu np. `answer-button-` wtedy `hovered` zamieni się na `answer-button-hovered`.